### DISCUSSION TITLE
Is there a build for MacOS?

### DISCUSSION THREAD

**Discussion by mikestan** [shark92651/LaserParamsConverter#15] [General]
I keep recommending your app to various forums and FB groups and have a handful of people who run MacOS that need a binary for their system. Any chance to get one added to the download page? I don't have a Mac anymore, so my instructions for using the source are just from googl'ing it. 🤔

**Comment by shark92651** [comment]
Currently the application targets .Net Framework 4.8. In order to create an executable that runs on both Windows and MacOS, I would need to rewrite it to target .NET Core which is cross-platform. I may have time to work on that in 2024, but cannot promise when it will be done.

**Reply by mreedmdev** [reply]
Hi, I am interested in contributing. A MacOS port in .NET Core. Is there a chance we could communicate about this, I can give you an email address or some other method you prefer?
